test/X25519KeyAgreementKey2020.spec.js

```diff
@@ -117,12 +119,17 @@ describe('X25519KeyAgreementKey2020', () => {
     });
 
     it('should export both public and private key', async () => {
-      const key = await X25519KeyAgreementKey2020.generate();
+      const key = await X25519KeyAgreementKey2020.generate({
+        controller: 'did:example:1234'
+      });
+      key.revoked = '2019-10-12T07:20:50.52Z';
```

Comment:
Please use the new one liner for RFC3339
https://github.com/digitalbazaar/http-signature-zcap-verify/blob/f3a100c3cfcff4cfde24614b61ab7c03b6f6efb8/tests/10-verify.spec.js#L178-L179

Reply:
Changed.